src/Explorer/Tree/ResourceTreeAdapter.tsx

```diff
@@ -351,6 +357,80 @@ export class ResourceTreeAdapter implements ReactAdapter {
     };
   }
 
+  private buildSchemaNode(collection: ViewModels.Collection): TreeNode {
+    if (collection.analyticalStorageTtl() == undefined) {
+      return null;
+    }
+
+    if (
+      collection.schema == undefined ||
```

Comment:
I think you can do:
`if (!collection.schema || !collection.schema.fields) { return undefined; }`